orion/evaluation/contextual.py

```diff
@@ -0,0 +1,206 @@
+""" The evaluator module provides metrics to assess the perfomance of
+anomaly detection methods using classification metrics: accuracy, precision,
+recall, and f1 score. This class addresses the problem as contextual anomalies.
+
+anomaly = [(start, end)] inclusive interval.
+"""
+
+from .common import _accuracy, _any_overlap, _f1_score, _precision, _recall, _weighted_segment
```

Comment:
We should always use explicit absolute imports:

```python
from orion.evaluation.common import ...
```

This helps readability by indicating where to find the code of the imports at any moment without forcing to look at which the current module is.

Reply:
Updated.